src/python/pants/option/global_options.py

```diff
@@ -1008,3 +1033,16 @@ def validate_instance(cls, opts):
                 "The --remote-store-initial-timeout option requires a positive number of "
                 "milliseconds."
             )
+
+        # Ensure that remote headers are ASCII (gRCP requirement).
```

Comment:
Typo: s/gRCP/gRPC/

Also, gRPC does support binary headers, although we don't currently have a way to set a binary header via the command-line. (Note that the Tonic enum for header keys has `KeyAndValueRef::Ascii` and `KeyAndValueRef::Binary` variants.) So the "as required by gRPC" part of the help message is technically wrong. Better to say that Pants only supports ASCII headers being set from the command-line.

Reply:
Cool, will fix in the followup.